aws/table_aws_sfn_state_machine_execution.go

```diff
@@ -0,0 +1,177 @@
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "execution_arn",
+				Description: "The Amazon Resource Name (ARN) that identifies the execution.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "status",
+				Description: "The current status of the execution.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "input",
+				Description: "The string that contains the JSON input data of the execution.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getStepFunctionsStateMachineExecution,
+			},
+			{
+				Name:        "output",
+				Description: "The JSON output data of the execution.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getStepFunctionsStateMachineExecution,
+			},
+			{
+				Name:        "start_date",
+				Description: "The date the execution started.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "state_machine_arn",
+				Description: "The Amazon Resource Name (ARN) of the executed state machine.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "stop_date",
+				Description: "If the execution already ended, the date the execution stopped.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "trace_header",
+				Description: "The AWS X-Ray trace header that was passed to the execution.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getStepFunctionsStateMachineExecution,
+			},
+			{
+				Name:        "input_details",
+				Description: "Provides details about execution input or output.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getStepFunctionsStateMachineExecution,
+			},
+			{
+				Name:        "output_details",
+				Description: "Provides details about execution input or output.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getStepFunctionsStateMachineExecution,
+			},
+
+			// Standard columns for all tables
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Name"),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("ExecutionArn").Transform(transform.EnsureStringArray),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listStepFunctionsStateMachineExecutions(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	// Create session
+	svc, err := StepFunctionsService(ctx, d)
+	if err != nil {
+		plugin.Logger(ctx).Error("listStepFunctionsStateMachineExecutions", "connection_error", err)
+		return nil, err
+	}
+
+	var arn string
+	if h.Item != nil {
+		arn = *h.Item.(*sfn.StateMachineListItem).StateMachineArn
+	} else {
+		arn = d.KeyColumnQuals["state_machine_arn"].GetStringValue()
```

Comment:
state_machine_arn is not a optional or required column, we already have used parentHydrate